ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1091,6 +1092,63 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
+            test_name: windows-containerd-with-ovs-e2e
+            node: 'antrea-windows-containerd-testbed'
+            description: 'This is the {test_name} test for {name}.'
+            branches:
+                - ${{sha1}}
+            builders:
+                - builder-e2e-win-containerd:
+                      e2e_test_type: 'windows-containerd-with-ovs-e2e'
+                      win_image_node: '{antrea_win_image_node_name}'
+            trigger_phrase: ^(?!Thanks for your PR).*/test-windows-(containerd-with-ovs-e2e|all).*
+            white_list_target_branches: [ ]
+            allow_whitelist_orgs_as_admins: true
+            admin_list: '{antrea_admin_list}'
+            org_list: '{antrea_org_list}'
+            white_list: '{antrea_white_list}'
+            only_trigger_phrase: true
+            trigger_permit_all: true
+            throttle_concurrent_builds_category:
+                - e2e-lock-per-testbed
+            throttle_concurrent_builds_enabled: 'true'
+            status_context: jenkins-windows-containerd-with-ovs-e2e
+            status_url: --none--
+            success_status: Build finished.
+            failure_status: Failed. Add comment /test-windows-containerd-with-ovs-e2e to re-trigger.
```

Comment:
Please update this job information in document. https://github.com/antrea-io/antrea/blob/main/ci/jenkins/README.md